package/Dockerfile

```diff
@@ -30,17 +30,17 @@ ENV CATTLE_HELM_VERSION v2.16.8-rancher1
 ENV CATTLE_K3S_VERSION v1.21.7+k3s1
 ENV CATTLE_MACHINE_VERSION v0.15.0-rancher73
 ENV CATTLE_MACHINE_PROVISION_IMAGE rancher/machine:${CATTLE_MACHINE_VERSION}
-ENV CATTLE_ETCD_VERSION v3.4.13
-ENV LOGLEVEL_VERSION v0.1.3
+ENV CATTLE_ETCD_VERSION v3.5.1
```

Comment:
same concern about kustomize above, bumping minor here and maybe we needed something in 3.4 that's gone. pls confirm we're ok for this bump

Reply:
Actually ETCD is already v3.5.1 in the release/v2.6 branch https://github.com/rancher/rancher/blob/release/v2.6/package/Dockerfile#L33 Not sure why we see this here. @rohitsakala is your fork up to date?

Reply:
Updated

Reply:
Updated.